### DISCUSSION TITLE
Thank You

### DISCUSSION THREAD

**Discussion by 58bits** [igordanchenko/yet-another-react-lightbox#237] [General]
Just wanted to say thank you for the really terrific project. 

Also amazing that you had the foresight to make the `title` and `description`  of the Captions plugin `ReactNode` - and not simple string values. In our case we serialize richtext from our CMS and assign that to the description property as a `ReactNode`. 

Once we had the types setup in our headless CMS, it was a breeze using your module to create the lightbox in our React / Next.js frontend.

Again - thank you for the effort, the amazing plugins, and the excellent docs.

**Comment by igordanchenko** [comment]
Thank you for your kind feedback! I'm glad to hear this library is making a difference.

**Comment by boly38** [comment]
Every project should have a `Thank You` discussion entry to let project-users gives some positive feedbacks because `issue` feature is often requirements/bugs/improvements.. and there is no (formal) real way to provide positive feedback out of github-repository-star ⭐ 

so I 100% agree with OP : I had to migrate from another slide show component, and this one is really easy to adopt, even try to add one or more plugin is a kid game. So I've no doubt about the contributors efforts to get this result. Great job 🎖️

**Reply by igordanchenko** [reply]
Thank you for your kind feedback! 🙏  I'll pin this discussion to make it easier for others to find.

**Comment by zhanziyang** [comment]
Thank you for this amazing component. It is so far the best I have ever used in React/Next. It simply works like a charm!

**Comment by matiduda** [comment]
Thank you, I don't think I've ever seen literally zero open issues on a popular GitHub repo, great work!

**Comment by nicksergeant** [comment]
@igordanchenko just another thank you here for an outstanding library that has great accessibility out of the box, and a lot of attention to detail to gestures and mobile UX. I've tried handfuls of lightbox libs over the years and yours wins _by far_. We're using it on dozens of websites at my agency.

I'd love to buy you a coffee and/or contribute some funds to the project if you make it possible to do so in the future (maybe with something like https://buymeacoffee.com/ ?)

Thanks again!
